### ISSUE TITLE
Not working on PHP 8.1

### ISSUE THREAD

**Issue by sammybammy52** [PHPOffice/PhpSpreadsheet#3320]
This is:

```
- [ x] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
install on php 8.1 for use

### What is the current behavior?
 Package phpoffice/phpspreadsheet has requirements incompatible with your PHP version, PHP exte  
  nsions and Composer version:                                                                    
    - phpoffice/phpspreadsheet 1.27.0 requires ext-gd * but it is not present.                    
    - phpoffice/phpspreadsheet 1.27.0 requires ext-zip * but it is not present. 

### What are the steps to reproduce?

Please provide a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve) of code that exhibits the issue without relying on an external Excel file or a web server:

```php
<?php

require __DIR__ . '/vendor/autoload.php';

// Create new Spreadsheet object
$spreadsheet = new \PhpOffice\PhpSpreadsheet\Spreadsheet();

// add code that show the issue here...
```
 Package phpoffice/phpspreadsheet has requirements incompatible with your PHP version, PHP exte  
  nsions and Composer version:                                                                    
    - phpoffice/phpspreadsheet 1.27.0 requires ext-gd * but it is not present.                    
    - phpoffice/phpspreadsheet 1.27.0 requires ext-zip * but it is not present. 
```
If this is an issue with reading a specific spreadsheet file, then it may be appropriate to provide a sample file that demonstrates the problem; but please keep it as small as possible, and sanitize any confidential information before uploading.

### What features do you think are causing the issue

- [x ] Reader
- [ ] Writer
- [ ] Styles
- [ ] Data Validations
- [ ] Formula Calculations
- [ ] Charts
- [ ] AutoFilter
- [ ] Form Elements

### Does an issue affect all spreadsheet file formats? If not, which formats are affected?

yes
### Which versions of PhpSpreadsheet and PHP are affected?
1.27

**Comment by MarkBaker** [comment]
So which of those two extensions aren't enabled for your installation of PHP?

Those requirements are not new; but they do need to be enabled for your PHP to run PhpSpreadsheet.

**Comment by sammybammy52** [comment]
> So which of those two extensions aren't enabled for your installation of PHP?
> 
> Those requirements are not new; but they do need to be enabled for your PHP to run PhpSpreadsheet.

I saw an instruction to uncomment extension=gd and extension=zip from my php.ini file, I did it for "extension=gd" but I didn't see "extension=zip" in the php.ini file

**Comment by MarkBaker** [comment]
PHP must be compiled [with zip support](https://www.php.net/manual/en/zip.installation.php)

**Comment by sammybammy52** [comment]
I solved it by installing php 8.1-gd and reinstalling it using the second command  below
`` sudo apt-get install php8.1-gd ``
`` composer require phpoffice/phpspreadsheet --ignore-platform-req=ext-gd ``
